baybe/parameters/numerical.py

```diff
@@ -79,12 +79,14 @@ def _validate_tolerance(  # noqa: DOC101, DOC103
     @property
     def values(self) -> tuple:  # noqa: D102
         # See base class.
-        return self._values
+        return tuple(DTypeFloatNumpy(itm) for itm in self._values)
```

Comment:
Asking without yet having a strong preference: is there a reason why you decided to cast in the property, instead of casting in the converter (i.e. when assigning)? I also guess that your way is probably the better alternative, but wanted to discuss nevertheless

Reply:
good point didnt think about that i think it would be somewhat nicer int he converter
is ti possible to do it like this `converter=lambda x: sorted(cattrs.structure(x, tuple[DTypeFloatNumpy, ...]))` ?

Reply:
In fact, there was a similar reason why I did not normalize the weights in the desirability objective during initialization, but rather kept the raw input. You remember?

Reply:
yes rememebr and it make sense, despite being prob a little more elegant thats an arg against the converter, so I can keep it and resolve?

Reply:
Yeah, let's keep it. Closing ...